kale/static_analysis/inspector.py

```diff
@@ -0,0 +1,263 @@
+import ast
```

Comment:
This file should not be included in the PR. It was delete in a previous PR. You might have included it during the merge by mistake